core/src/main/java/com/linecorp/armeria/client/cookie/CookiePolicy.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client.cookie;
+
+import static java.util.Objects.requireNonNull;
+
+import java.net.URI;
+
+import javax.annotation.Nullable;
+
+import com.linecorp.armeria.common.Cookie;
+import com.linecorp.armeria.internal.client.PublicSuffix;
+
+import io.netty.util.NetUtil;
+
+/**
+ * Specifies how {@link CookieJar} should accept new {@link Cookie}.
+ */
+@FunctionalInterface
+public interface CookiePolicy {
+
+    /**
+     * Creates a {@link CookiePolicy} that accepts all cookies.
+     */
+    static CookiePolicy acceptAll() {
+        return (uri, cookie) -> true;
+    }
+
+    /**
+     * Creates a {@link CookiePolicy} that rejects all cookies.
+     */
+    static CookiePolicy acceptNone() {
+        return (uri, cookie) -> false;
+    }
+
+    /**
+     * Creates a {@link CookiePolicy} that only accepts cookies from the original server.
+     */
+    static CookiePolicy acceptOriginalServer() {
```

Comment:
I'm not sure which sounds better but how about `acceptOriginOnly()`? Which one sounds better?

Reply:
`acceptOriginOnly` is nice, consistent with `all`, `none`, and `only`